### PR TITLE
Merge Front End to Main

### PR DESCRIPTION
**Pull request by Jm-altar**
### Type

why type of pr is this for?

-Very Big Update for Front End.

### Summary

- Contains all of the front end components and utilities.

### Details

-Pet Sprites
-UI Panel for user
-Header
-Stage Design and animations
- Added Functional Utilities for health, happiness and hunger
-
## Notes

- Health Bar animation not yet working.
- Moods panel also not yet working.
- Pet choosing not yet implemented.
- Back end not yet connected.

### CODE REVIEW

File: client/src/App.tsx
Comment (kenchyyy) [inline client/src/App.tsx:1]: Are the deleted lines unnecessary?

**Review by itsantonle — APPROVED** [review]

**Review by kenchyyy — COMMENTED** [review]
Rest are good

**Review by kenchyyy — APPROVED** [review]
Looks good

File: client/src/App.tsx
Comment (kenchyyy) [reply client/src/App.tsx:1]: ok na ni

**Pull request closed without merging**